### DISCUSSION TITLE
Block Ads on youtube.com

### DISCUSSION THREAD

**Discussion by Daniel-from-Germany** [AdguardTeam/AdGuardHome#5538] [Q&A]
Hello, is there a way to block ads on youtube.com?
I must add the domains to the userdefine filters to use this sites :-(

@@gmail.com
@@google.com
@@youtube.com

**Comment by ghost** [comment]
Nope. YouTube, Facebook, and some other big tech companies now integrate ads into actual playback media content. The only way to get rid of such ads is through extensions like uBlock Origin. AdGuard Home can't do that. It blocks only on domain level.
